packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -461,6 +461,10 @@ abstract class State<T extends StatefulWidget> {
   /// The given build context object contains information about the location in
   /// the tree at which this widget is being built. For example, the context
   /// provides the set of inherited widgets for this location in the tree.
+  ///
+  /// The context argument is always the same as [context]. This argument is
```

Comment:
`[context]` should be `[State.context]` to reference the getter instead of param.